lib/reports/resolvers.ts

```diff
@@ -361,34 +360,47 @@ export const generateAccountantReport: Resolver<
     _count: {
       paymentMethod: true,
     },
-  });
+  };
 
-  const totalAggregate = await prisma.application.aggregate({
-    where: {
-      createdAt: {
-        gte: startDate,
-        lt: endDate,
-      },
-    },
-    _sum: {
-      processingFee: true,
-      donationAmount: true,
-    },
-    _count: {
-      paymentMethod: true,
-    },
-  });
+  const methodPaymentRequest = JSON.parse(JSON.stringify(paymentRequest));
```

Comment:
1. I might be missing something but doesn't `JSON.parse(JSON.stringify(x))` == `x`? Is the intention to make a deep copy, in which case I'd suggest defining a `deepCopy` function?

2. Nit for naming
```suggestion
  const paymentGroupByMethodQuery = JSON.parse(JSON.stringify(paymentRequest));
```